From 8bc60f9c097965e17f1cfa32e66461aacdb11083 Mon Sep 17 00:00:00 2001
Message-Id: <8bc60f9c097965e17f1cfa32e66461aacdb11083.1387385974.git.minovotn@redhat.com>
In-Reply-To: <7cba796b16325e96d01e65bd265b4073ab103157.1387385974.git.minovotn@redhat.com>
References: <7cba796b16325e96d01e65bd265b4073ab103157.1387385974.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 17 Dec 2013 06:46:39 +0100
Subject: [PATCH 5/5] HMP: Disable drive_add for Red Hat Enterprise Linux

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1387262799-10350-7-git-send-email-armbru@redhat.com>
Patchwork-id: 56297
O-Subject: [PATCH v2 6/6] HMP: Disable drive_add for Red Hat Enterprise Linux
Bugzilla: 889051
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Same as in RHEL-6; commit ed45bb8 there.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 hmp-commands.hx | 2 ++
 1 file changed, 2 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hmp-commands.hx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index bfb565b..32cc33c 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1072,6 +1072,7 @@ using the specified target.
 ETEXI
 #endif
 
+#if 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "drive_add",
         .args_type  = "pci_addr:s,opts:s",
@@ -1084,6 +1085,7 @@ ETEXI
         .help       = "add drive to PCI storage controller",
         .mhandler.cmd = drive_hot_add,
     },
+#endif
 
 STEXI
 @item drive_add
-- 
1.7.11.7

